src/enterprise.hbs

```diff
@@ -0,0 +1,65 @@
+---
+title: Enterprise
+description: Give your students the grade tracking resource they deserve
+hideHeaderLogoText: false
+---
+
+{{#block "hero"}}
+<div class="container-sm">
+	<div class="hero-inner">
+		<div class="hero-copy">
+			<h1 class="gb-header hero-paragraph">Partner with the only <br> student-centered grade tracker</h1>
+			{{!-- @todo: no inline styles! --}}
+			<p class="hero-paragraph" style="font-size: 20px">Expert solutions and support for schools and groups of all sizes</p>
+		</div>
+		<div class="hero-cta">
+			<a class="button button-primary button-wide-mobile"
+				href="{{site.url}}/enterprise">Get in touch</a>
+		</div>
+	</div>
+</div>
+{{/block}}
+
+<section class="image-block section">
+	<div class="image-block-inner section-inner">
+		<div class="image-block-wrap">
+			<div class="container">
+				<div class="image-block-item">
+					<div class="image-block-item-image is-revealing">
+						<img src="/static/images/undraw_app_data_516s.svg" alt="Viewing statistics">
+					</div>
+					<div class="image-block-item-body is-revealing">
+						<h3 class="mt-0">Better insights into academic progress</h3>
+						<p class="mb-0">Our powerful prediction and statistics tools allow your students to focus on what's most
+							important, and spend less time worrying about their grades.
```

Comment:
```suggestion
						<p class="mb-0">
							Our powerful prediction and statistics tools allow your students to focus on what's most	important, and spend less time worrying about their grades.
```

Removes accidental layout issues caused by whitespace.

Also, as a general rule of thumb, I try not to cut copy w/ line breaks - if you're trying to edit it, enable word wrapping, and if you're not, disabling wrapping will put it out of sight.